invenio_rdm_records/fixtures/data/vocabularies.yaml

```diff
@@ -25,9 +25,6 @@ relationtypes:
 # 'subjects' type needs to be created to prevent querying errors in the facets
 # but they are highly dependent on the kind of InvenioRDM instance, so no
 # defaults are provided
-subjects:
```

Comment:
To be discussed. Maybe since you are working/have more experience with the loading, you might have some insights @fenekku